src/SeaIceDynamics/SeaIceDynamics.jl

```diff
@@ -0,0 +1,28 @@
+module SeaIceDynamics
+
+using ClimaSeaIce
+
+using Oceananigans
+using KernelAbstractions: @kernel, @index
+using Oceananigans.Utils: launch!
+using Oceananigans.Operators
+using Oceananigans.Grids
+using Oceananigans.Grids: architecture
+
+## A Framework to solve for the ice momentum equation, in the form:
+## 
+##     ∂u                   τₒ    τₐ
+##     -- + f x u = ∇ ⋅ σ + --  + -- + g∇η
```

Comment:
not sure what the importance of `g∇η` is, but it might be better to leave it off for the time being